app/views/admin/feedback_messages/_feedback_message.html.erb

```diff
@@ -90,7 +90,7 @@
             <% if feedback_message_emails.any? %>
               <% feedback_message_emails.each do |email| %>
                 <div class="email__container">
-                  <p class="to__subject">Type: <%= email.utm_campaign.capitalize %></p>
+                  <p class="to__subject">Type: <%= email&.utm_campaign&.capitalize %></p>
```

Comment:
I think you technically need the `&` only between `utm_campaign` and `capitalize` since `email` can't be `nil` but I think we can live with it :)

Reply:
Agreed!